.github/workflows/pipeline.yaml

```
@@ -0,0 +1,53 @@
+        path: node_modules
+        key: ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-
+    - name: Installing dependencies
+      run: yarn install --frozen-lockfile --prefer-offline
+      if: |
+        steps.cache-yarn-cache.outputs.cache-hit != 'true' ||
+        steps.cache-node-modules.outputs.cache-hit != 'true'
+
+    - name: test
+      run: yarn test
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)